src/workspace-data/Data.js

```diff
@@ -712,6 +713,21 @@ export const WorkspaceData = _.flow(
       [signal]
     );
 
+    const loadWdsCapabilities = useCallback(
+      (url) => {
+        return Ajax(signal)
+          .WorkspaceData.getCapabilities(url)
+          .then((capabilitiesResult) => {
+            setWdsCapabilities({ status: 'Ready', state: capabilitiesResult });
+          })
+          .catch((error) => {
+            setWdsCapabilities({ status: 'Error', state: 'Error loading WDS capabilities' });
+            reportError('Error loading WDS capabilities', error);
+          });
+      },
```

Comment:
Existing Terra UI code mostly uses async/await vs Promise methods.

```suggestion
      async (url) => {
        try {
          capabilitiesResult = await Ajax(signal).WorkspaceData.getCapabilities(url);
          setWdsCapabilities({ status: 'Ready', state: capabilitiesResult });
        } catch (error) {
          setWdsCapabilities({ status: 'Error', state: 'Error loading WDS capabilities' });
          reportError('Error loading WDS capabilities', error);
        }
      },
```

Reply:
Thanks, this felt like I was going the wrong direction but I couldn't figure out how to get back on the path.

Fixed in f95bcee32e05964c298987cfa9f9da34006e659e